python/GangaLHCb/__init__.py

```diff
@@ -85,43 +85,21 @@ def _store_root_version():
 
 
 def _store_dirac_environment():
+    from GangaDirac.Lib.Utilities.DiracUtilities import write_env_cache
     diracversion = _guess_version('LHCBDIRAC')
-    import tempfile
-    import subprocess
-    import os
     platform = os.environ['CMTOPT']
     setup_script = 'SetupProject.sh'
```

Comment:
Sure, what would be the full `lb-run` command needed to get the environment? I'll give it a test.